### ISSUE TITLE
Improve lore2sphinx buildbot results for `projects/core/howto/logging.xhtml`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4864]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4864|
|Type|enhancement|
|Created|2011-02-11 05:11:04Z|
|Branch|https://github.com/twisted/twisted/tree/logging-sphinx-4864|

like [#4566](https://github.com/twisted/twisted/issues/4566), except for the logging howto

<details><summary>Searchable metadata</summary>

```
trac-id__4864 4864
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__Lore_to_Sphinx Lore to Sphinx
branch__branches_logging_sphinx_4864 branches/logging-sphinx-4864
branch_author__screwtape__jdb screwtape, jdb
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1297401064000000 1297401064000000
changetime__1300137320000000 1300137320000000
version__None None
owner__lvh lvh

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [30660]) Branching to 'logging-sphinx-4864'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [30661]) Merge the logging parts of the impossibly huge [[#4568](https://github.com/twisted/twisted/issues/4568)](https://github.com/twisted/twisted/issues/4568) branch


refs #4568
refs [#4864](https://github.com/twisted/twisted/issues/4864)

**Comment by twisted-trac** [comment]
|[<img alt="tenth10th's avatar" src="https://avatars.githubusercontent.com/u/854235?s=50" width="50" height="50">](https://github.com/tenth10th)<a name="note_4"></a>|@tenth10th set owner to Screwtape|
|-|-|

For the first change: If you're removing the link on "msg" because the link is broken, that's due to a bug in the API generation (see [#11366](https://github.com/twisted/twisted/issues/11366)) and it should be be possible to fix it, rather than removing the link altogether.

The second change, where you add an example that imports DailyLogFile: The example is fine, but it would be nice if DailyLogFile was explained before being used in example code. (Maybe move the new example down so that it's after the description of DailyLogFile under the "Log Files" heading, or vice-versa?)

Finally, there are several places where a space appears between the end of a markup tag and the ending punctuation (usually a period). For example:

```
"You can disable this by passing setStdout=False to startLogging ."
```

If this is intentional, great! Or if it's required by Sphinx, I guess we're stuck with it. But if it's by accident, maybe we need to handle those markup cases differently?

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_6"></a>|@khorn commented|
|-|-|

The space between the end of a markup tag and punctuation is an artifact of the lore2sphinx script which converts the lore sources.  It can be fixed in the Sphinx sources, but doing it automatically and reliably is a total beast, so we just stick a space everywhere for now.

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_7"></a>|@khorn commented|
|-|-|

(In [31054]) Fix broken api link to t.p.l.LogPublisher.msg

refs: [#4864](https://github.com/twisted/twisted/issues/4864)

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_8"></a>|@khorn removed owner|
|-|-|

send to review queue

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)|@lvh set owner to @lvh<br>@lvh set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_10"></a>|@lvh commented|
|-|-|

Yeah pretty trivial fix. merging. Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_11"></a>|@lvh set status to `closed`|
|-|-|

(In [31058]) Better lore2sphinx output for logging.xhtml

Author: khorn, Screwtape, jdb
Reviewer: tenth, lvh
Fixes: [#4864](https://github.com/twisted/twisted/issues/4864)

"Make Sphinx look gooder" -- JP
